src/configure/helper/AssetHandler.ts

```diff
@@ -17,7 +17,7 @@ const Layer = "AssetCreationHandler";
 export class AssetHandler {
     // tslint:disable-next-line:no-reserved-keywords
     public async createAssets(assets: TemplateAsset[], inputs: WizardInputs, createAsset: (name: string, type: TemplateAssetType, data: any, inputs: WizardInputs) => Promise<string>): Promise<void> {
-        if (inputs.pipelineConfiguration.template.label === "Containerized application to AKS") {
+        if (inputs.pipelineConfiguration.template.label === "Containerized application to Azure Kubernetes Service (AKS)") {
```

Comment:
Won't it be better if we specify this as a constant somewhere, so that reusing this string at multiple places becomes easier? This will also prevent mistype issues.